src/onboarding-screen/OnboardingScreenComponent.tsx

```diff
@@ -48,14 +50,16 @@ export const OnboardingScreenComponent = ({
       footerDescription={footerDescription}
     >
       <View style={styles.header}>{illustration}</View>
-      <ThemeText
-        type="body__primary--big--bold"
-        color={themeColor}
-        style={styles.title}
-        accessibilityRole="header"
-      >
-        {title}
-      </ThemeText>
+      <View ref={focusRef}>
```

Comment:
Needs `accessible` to work on Android

Reply:
Yes. Would be practical with a lint rule for this if possible 😅

Reply:
Fixed in https://github.com/AtB-AS/mittatb-app/pull/4234#issue-2083914214